hoomd/md/ForceCompositeGPU.cc

```diff
@@ -267,9 +267,18 @@ void ForceCompositeGPU::updateCompositeParticles(uint64_t timestep)
     ArrayHandle<Scalar4> d_postype(m_pdata->getPositions(),
                                    access_location::device,
                                    access_mode::readwrite);
+    ArrayHandle<Scalar4> d_velocity(m_pdata->getVelocities(),
+                                    access_location::host,
```

Comment:
These all need to have `access_location::device`. I'm confused about how these unit tests could pass for the GPU, as this should produce a segfault.

Reply:
There are a few other pointers that use `access_location::host` but I haven't touched them. They're h_body, h_tag, and h_flag. Should those be d_body, d_tag, and d_flag? (I'm assuming the letter in front determines whether `access_location` should be host or device)